i3pystatus/openvpn/systemd.py

```diff
@@ -1,12 +1,12 @@
-from i3pystatus import IntervalModule
+from i3pystatus.openvpn import Backend
```

Comment:
This throws an import error for me. Should `Systemd` subclass `SettingsBase` like the `File` module?